src/AppBundle/Mailjet/Message/CommitteeContactMembersMessage.php

```diff
@@ -23,7 +23,8 @@ public static function create(array $recipients, Adherent $animator, string $mes
             $animator->getFullName(),
             "L'animateur d'un comité que vous suivez vous a envoyé un message",
             ['animator_firstname' => $animator->getFirstName(), 'animator_message' => $message],
-            ['target_firstname' => $animator->getFirstName()]
+            ['target_firstname' => $animator->getFirstName()],
+            Uuid::uuid4()
```

Comment:
That would be better if we could know the uuid in advance, especially when we'll have to write the functional tests for this. But I don't see how we could have such a value in this situation...